src/code/z_play.c

```diff
@@ -609,9 +609,9 @@ void Play_Update(PlayState* this) {
                     }
 
                     if (this->transitionTrigger == TRANS_TRIGGER_END) {
-                        this->transitionCtx.setType(&this->transitionCtx.instanceData, 1);
+                        this->transitionCtx.setType(&this->transitionCtx.instanceData, TRANS_TYPE_FADE_IN);
                     } else {
-                        this->transitionCtx.setType(&this->transitionCtx.instanceData, 2);
+                        this->transitionCtx.setType(&this->transitionCtx.instanceData, TRANS_TYPE_FADE_OUT);
```

Comment:
So is "fade in" the transition fading in (e.g. screen fades to black) or the screen fading in (e.g. screen fades from black)

From context here I'd guess "fade in" = "screen fades from black", which is the opposite of what I expected <img src="https://cdn.discordapp.com/emojis/1021127774670438431.webp?size=240&quality=lossless" alt="PepeLaugh" height="24">

(we had the same issue naming stuff in the transition system)

fill/unfill like in `SANDSTORM_FILL`/unfill may be an idea

Reply:
We could use the well defined standard that movies have been using for around a 100 years, but I remember the last time I suggested doing this we decided to do the complete opposite.

Reply:
When naming these I was definitely wasn't really thinking of the transition as its own entity, so everything is named based on gameplay. So Fade-In would mean fade from black screen back into gameplay.

Went searching through discord and found the discussion (https://discord.com/channels/688807550715560050/838852326231769089/963598865892802642). Perhaps just using the same terminology decided there (start and end)?

Reply:
I guess start/end works too

I would have liked the names to be more "drawing focused", since start/end for trans_trigger also refers to some logic to be run, but start/end works fine here too